src/Airport.js

```diff
@@ -0,0 +1,38 @@
+const Airplane = require(`./Airplane`);
+
+class Airport {
+
+    constructor() {
+    }
+
+    capacity = 10;
+    airportPlanes = [];
+    allLandedPlanes = [];
+
+
+
+    landPlane = (weather, plane) => {
+        if (!this.airportPlanes.includes(plane) && this.airportPlanes.length < this.capacity && weather < 0.85 && plane instanceof Airplane && !this.allLandedPlanes.includes(plane)) {
```

Comment:
Arguably, for clarity reasons it might have been better to break this single conditional statement that is made up of multiple conditions into a few different conditional statements because at the moment the print output doesn't provide the user with precise information but rather a list of possible reasons for failure. On the other hand, however, you could argue that this is what the testing is designed to clarify...